Directory.Build.props

```diff
@@ -297,7 +293,6 @@
 
     <BaseOutputPath Condition="'$(BaseOutputPath)'==''">$(BinDir)</BaseOutputPath>
 
-    <OutputPathSubfolder Condition="'$(IsCompatAssembly)'=='true'">/Compat</OutputPathSubfolder>
```

Comment:
for context, did you remove that line as we don't deal with compat assemblies anymore?

Reply:
Correct, those aren’t used anymore.